File: lib-mvpbase-appcompat/src/main/java/com/baozi/mvp/templet/helper/MDToolBarHelperImpl.java
package com.baozi.mvp.templet.helper;

import android.view.View;

import com.baozi.mvp.templet.options.ToolbarOptions;
import com.baozi.mvp.view.ToolbarView;

/**
 * @author jlanglang  2017/3/7 16:29
 * @版本 2.0
 * @Change
 */
public class MDToolBarHelperImpl extends BaseToolBarHelperImpl {

    public MDToolBarHelperImpl(ToolbarView uiView, View rootView, int toolbarLayout) {
        super(uiView, rootView, toolbarLayout);
    }

    @Override
    public void initToolbar() {
        ToolbarHelper.SimpleInitToolbar(mToolbarView.getContext(), mToolbar, true);
    }

    @Override
    public void setToolbarOptions(ToolbarOptions options) {
        super.setToolbarOptions(options);

    }

    @Override
    public void setTextSize(int size) {

    }

    @Override
    public void setTitleSize(int size) {

    }
}

File: lib-mvpbase-kotlin/src/main/java/com/linfeng/mvp/templet/helper/MDToolBarHelperImpl.kt
package com.linfeng.mvp.templet.helper

import android.view.View

import com.linfeng.mvp.templet.options.ToolbarOptions
import com.linfeng.mvp.view.ToolbarView

/**
 * @author jlanglang  2017/3/7 16:29
 * @版本 2.0
 * @Change
 */
class MDToolBarHelperImpl(uiView: ToolbarView, rootView: View, toolbarLayout: Int) : BaseToolBarHelperImpl(uiView, rootView, toolbarLayout) {

    override fun initToolbar() {
        ToolbarHelper.simpleInitToolbar(mToolbarView.mContext, toolbar, true)
    }

    override fun setToolbarOptions(toolbarOptions: ToolbarOptions) {
        super.setToolbarOptions(toolbarOptions)

    }

    override fun setTextSize(size: Int) {

    }

    override fun setTitleSize(size: Int) {

    }
}
